Test/git-issues/git-issue-1514.dfy

```diff
@@ -0,0 +1,31 @@
+// RUN: %dafny /compile:3 /rprint:"%t.rprint" "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+datatype Option<+T> = None | Some(value: T) {
+  predicate method IsFailure() {
+    None?
+  }
+  function method PropagateFailure<U>(): Option<U>
+    requires None?
+  {
+    None
+  }
+  function method Extract(): T
+    requires Some?
+  {
+    value
+  }
+}
```

Comment:
```
include "../libraries/src/Wrappers.dfy"
import opened Wrappers
```
This should do the trick, right?

Reply:
That's a good idea. I never saw that the library was imported within the Test folder itself.